robot/impl/local_robot_test.go

```diff
@@ -394,42 +389,35 @@ func TestConfigRemoteWithAuth(t *testing.T) {
 			var r2 robot.LocalRobot
 			if tc.Managed {
 				remoteConfig.Remotes[0].Auth.Entity = "wrong"
-				_, shutdown = initTestRobot(t, context.Background(), remoteConfig, logger)
-				defer shutdown()
+				robotimpltest.LocalRobot(t, context.Background(), remoteConfig, logger)
 
 				remoteConfig.AllowInsecureCreds = true
 
-				r3, shutdown := initTestRobot(t, context.Background(), remoteConfig, logger)
-				defer shutdown()
+				r3 := robotimpltest.LocalRobot(t, context.Background(), remoteConfig, logger)
 				remoteBot, ok := r3.RemoteByName("foo")
 				test.That(t, ok, test.ShouldBeFalse)
 				test.That(t, remoteBot, test.ShouldBeNil)
 
 				remoteConfig.Remotes[0].Auth.Entity = entityName
 				remoteConfig.Remotes[1].Auth.Entity = entityName
-				_, shutdown = initTestRobot(t, context.Background(), remoteConfig, logger)
-				shutdown()
```

Comment:
we have some tests like this where we initialize a local robot and immediately close it. the new version of this test line captures the spirit of the assertion more explicitly IMO, since it's not obvious what `shutdown` does without visiting the helper.